src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,251 @@
+  But although https://pub.dev/packages/sdk_builds exists,
+  I think it's just referring to the Dart SDK.
+  So we can't say everything's a package, can we?]
+</span>
+
+[`pub deps`]: /tools/pub/cmd/pub-deps)
+[`pub get`]: /tools/pub/cmd/pub-get)
+[`pub upgrade`]: /tools/pub/cmd/pub-upgrade)
+
+
+## Using the output
+
+As a reminder, here's what the output of `pub outdated` looks like:
+
+
+```terminal
+$ pub outdated
+Package              Current   Upgradable  Resolvable  Latest    
+dependencies: all up-to-date
+
+dev_dependencies    
+build_web_compilers  1.2.3     1.2.3       2.10.0      2.10.0    
+
+transitive dependencies
+_fe_analyzer_shared  -         -           2.0.0       2.0.0     
+analyzer             0.35.4    0.35.4      0.39.5      0.39.5    
+...
+
+9  dependencies are constrained to versions that are older than a resolvable version.
+To update these dependencies, edit pubspec.yaml.
+```
+
+As a rule, update the **dependencies** and **dev_dependencies** sections
+of your `pubspec.yaml` file
+to depend on the **resolvable** versions of out-of-date packages.
+<span style="color:red">
+  [PENDING: correct? explain why?]
+</span>
+
+Here are the meanings of the version numbers in the output:
+
+Current
+: The version used in your package, as recorded in `pubspec.lock`.
+  If the package isn't in `pubspec.lock`,
+  the value is `-`.
+  <span style="color:red">
+    [PENDING: I'm curious about why the package isn't in pubspec.lock...
+    but maybe it's OK not to talk about that.]
+  </span>
+
+Upgradable
+: The latest version allowed by your `pubspec.yaml` file.
+  This the version that `pub upgrade` resolves to.
+  <span style="color:red">
+    [PENDING: When is this `-`? If Current is `-`?]
+  </span>
+
+Resolvable
+: The latest version that can be resolved.
+  This version corresponds to what you get if
+  you set all version constraints in `pubspec.yaml` to `any`
+  and then run `pub upgrade`.
+  <span style="color:red">
+    [PENDING: why might this be `-`? It seems like it happens
+    when the package won't be used any more once you update the
+    pubspec.]
+  </span>
+
+{{site.alert.warning}}
+  We don't recommend using `any` as a version constraint,
+  for performance reasons.
+{{site.alert.end}}
+
+Latest
+: The latest version of the package available,
+  excluding prereleases unless you use the option `--prereleases`.
+  <span style="color:red">
+    [PENDING: This will never be `-`, right?]
+  </span>
+
+For example, say your app depends on the `foo` and `bar` packages,
+but the latest version of `bar` depends on an old major version of `foo`.
+The result is that the latest _resolvable_ version of `foo`
+is different from the _latest_ version of `foo`.
+
+
+## Options
+
+For options that apply to all pub commands, see
+[Global options](/tools/pub/cmd#global-options).
+
+`--json`
+: Use this option to generate output in JSON format.
+
+`--[no-]color`
+: Use this option to change whether the output has multiple colors.
```

Comment:
Suggest s/has multiple colors/has color/, as white isn't a color, at least in physics ;-)

Reply:
Unless you're talking light, as in computer screens. :)

Reply:
Maybe 'is colorized'?

Reply:
Maybe "uses color for emphasis"?